module/Application/src/Mapper/BaseMapper.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Application\Mapper;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\OptimisticLockException;
+use Doctrine\ORM\ORMException;
+
+abstract class BaseMapper
+{
+    /**
+     * Doctrine entity manager.
+     *
+     * @var EntityManager
+     */
+    protected EntityManager $em;
+
+    /**
+     * Constructor.
+     */
+    public function __construct(EntityManager $em)
+    {
+        $this->em = $em;
+    }
+
+    /**
+     * @param object $entity
+     * @throws ORMException
+     */
+    public function persist(object $entity) {
+        $this->em->persist($entity);
+        $this->em->flush();
+    }
```

Comment:
I would add the return type `void` here, like:

```php
public function persist(object $entity): void {
    $this->em->persist($entity);
    $this->em->flush();
}
```

This also applies to `remove()`, `removeById()`, `flush()`, and `detach()`.